src/hooks/useEditor.ts

```diff
@@ -81,9 +81,7 @@ export class ReactEditorView extends EditorView {
     super(place, {
       state: EditorState.create({
```

Comment:
Question... should we pass an empty schema here, too? Is there a risk that ProseMirror creates some DOM because the schema requires some non-empty doc by its content rules?